recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {#% set version = datetime.datetime.utcnow().strftime('%Y.%m.%d.%H.%M.%S') %#}
-{% set version = '2020.03.30' %}
+{% set version = '2020.04.01' %}
```

Comment:
This is yesterday. Let's just go with today?

Reply:
Sorry, was certain that I wouldn't make a mistake in such a simple PR :(

Reply:
Hopefully, we have grpc-cpp migration later, then I'll fix this. 🤞